.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: ci
+
+on:
+   workflow_dispatch:
+   schedule:
+     - cron: 0 0 1 * 0     # monthly
+   pull_request:
+     branches:
+       - main
+       - development
+
+jobs:
+  tox:
+    strategy:
+      matrix:
+        os: [ubuntu-latest] #[ubuntu-latest, macos-latest, windows-latest]
+        python-version: ['3.10'] #['3.8', '3.9', '3.10']
```

Comment:
just curious: we'll want to test multiple versions and OS eventually -- are you thinking we just wait to do that until a bit further along?

Reply:
yes, exactly. for now, since it's early days I am testing on single os